help-beta/.gitignore

```diff
@@ -0,0 +1,23 @@
+# build output
+dist/
+# generated types
+.astro/
+
+# dependencies
+node_modules/
+
+# logs
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+pnpm-debug.log*
+
+*.mdx
```

Comment:
It may prove annoying when switching between branches that have this built that the `.gitignore` isn't present in older versions. Not much to be done about it.

Reply:
I tested and it just has `help-beta/` in `git status`, about the best one can hope for.